packages/gatsby-plugin-graphql-codegen/package.json

```diff
@@ -46,7 +47,11 @@
     "@types/node": "^12.0.8",
     "gatsby": "^2.9.2",
     "graphql": "^14.6.0",
-    "typescript": "^3.7.5"
+    "ts-loader": "^7.0.0",
+    "typescript": "^3.7.5",
+    "webpack": "^4.42.1",
```

Comment:
`gatsby-node` has to be placed in the root, otherwise Gatsby can't find it. Because of this, all files inside `src` is compiled to the root dir. I didn't want to ship unnecessary files, so I've been manually add/remove files to `.npmignore`. This is quite cumbersome as the amount of files increase.

Rollup was no.1 choice, but for whatever reasons I couldn't make it work with various typescript plugins. I had little time, so I decided to go with webpack, which worked first try.

Now come to think about it, I've realized I don't really need to bundle, I can just npmignore the src dir 🤦‍♀️ Or just ship the whole thing.